### PR TITLE
Added comments indent.

### PR DESCRIPTION
**Pull request by k82cn**
Added comments indent.

### CODE REVIEW

File: myriad-executor/src/main/java/org/apache/myriad/executor/MyriadExecutorAuxService.java [hunk 1]
```diff
@@ -50,7 +50,7 @@
   private Thread myriadExecutorThread;
   // Storing container id strings as it is difficult to get access to
   // NodeManager's NMContext object from an auxiliary service.
-  private Set<String> containerIds = new HashSet<>();
+  private Set<String> containerIds = new HashSet<String>();
```
Comment (DarinJ) [inline myriad-executor/src/main/java/org/apache/myriad/executor/MyriadExecutorAuxService.java]: From what I've seen most people are preferring to use <> on the RHS as it's pretty redundant.  Not sure I think we should change these.

File: myriad-scheduler/src/main/java/org/apache/myriad/Main.java [hunk 2]
```diff
@@ -80,17 +80,17 @@
...
+   *
+   *  1. MyriadDriverManager
+   *  2. MyriadWebServer
+   *  3. TaskTerminator
+   *  4. HealthCheckRegistry
+   *
+   *  Main uses the Guice Injector framework to manage the Myriad object graph and is
+   *  configured by myriad-config-default.yml
+   */
   public static void initialize(Configuration hadoopConf, AbstractYarnScheduler yarnScheduler, RMContext rmContext,
```
Comment (DarinJ) [inline myriad-scheduler/src/main/java/org/apache/myriad/Main.java]: Good catch I'd never notice that.

**Comment by DarinJ** [comment]
@k82cn would happily accept a PR for the comment fixes but do wish to change the initialization as you suggested.  If we're being inconsistent through out let's go ahead and change everything the <> on the RHS (another possible PR).

**Comment by k82cn** [comment]
Some of indents are fixed in https://github.com/apache/incubator-myriad/commit/577c30b1abdaa30313251c887c014aebac3fd93c . Only update one line.

@DarinJ , agree to commit indent firstly. For `<>` part, we can update it later if necessary.

**Comment by DarinJ** [comment]
@k82cn great will merge today/tomorrow depending on time.

**Comment by k82cn** [comment]
@DarinJ , thanks :).

**Pull request closed without merging**